src/services/createRouterRoute.ts

```diff
@@ -3,11 +3,18 @@ import { RouterPush, RouterPushOptions } from '@/types/routerPush'
 import { RouteUpdate } from '@/types/routeUpdate'
 import { Writable } from '@/types/utilities'
 
+const isRouterRouteSymbol = Symbol('isRouterRouteSymbol')
```

Comment:
I needed a way to type guard the RouterRoute. This works but I'd like to revisit this. The proxy based approach is going to get more and more complex and fragile as we add stuff to this type.

Reply:
addressed in #199